src/common/components/repository-row/index.js

```diff
@@ -65,8 +70,21 @@ class RepositoryRow extends Component {
       <Dropdown>
         <Row>
           <Data col="100">
-            This repo needs a snapcraft.yaml file, so that Snapcraft can make it buildable, installable, and runnable.
-            {/* TODO: add more info/links as in spec */}
+            <p>
+              This repo needs a snapcraft.yaml file,
+              so that Snapcraft can make it buildable,
+              installable, and runnable.
+            </p>
+            <p>
+              <a href={ LEARN_THE_BASICS_LINK } target="_blank">Learn the basics</a>,
```

Comment:
We may need them with 'external link' icon, but we don't have this style globally yet (just in components that used that before...)